[IMP] accounting: add info on partial payment for vendor bills (#10674)
odoo/documentation: content/applications/finance/accounting/vendor_bills.rst

> Upon payment of the vendor bill, click on :guilabel:`Register Payment`. A new window pops up.

Reviewer: ```suggestion
Upon payment of the vendor bill, click on :guilabel:`Register Payment` to open a new payment window.
```
I know this wasn't your sentence originally, but it sounds awkward so let's change it :)

Select the :guilabel:`Journal`, the :guilabel:`Payment Method`, the :guilabel:`Amount` you wish to
pay (full or partial payment), and the :guilabel:`Currency`. In the case of a partial payment (when
the :guilabel:`Amount` paid is less than the total remaining amount on the vendor bill), the
:guilabel:`Payment Difference` field displays the outstanding balance.
You have two options:

> - :guilabel:`Keep open` : to keep the bill open and mark it with a :guilabel:`Partial` banner;

Reviewer: ```suggestion
- :guilabel:`Keep open`: to keep the bill open and mark it with a :guilabel:`Partial` banner;
```

> - :guilabel:`Mark as fully paid`. In this case, select an account in the

Reviewer: ```suggestion
- :guilabel:`Mark as fully paid`: In this case, select an account in the
```

  :guilabel:`Post Difference In` field and change the :guilabel:`Label` if needed. Odoo will create
  a journal entry balancing the account receivable with whichever account is selected.

.. image:: vendor_bills/partial-payment.png
   :alt: register a partial payment

> Odoo fills the :guilabel:`Memo` field

Reviewer: ```suggestion
Odoo fills the :guilabel:`Memo` field automatically if the :guilabel:`Payment Reference` has been
set correctly in the vendor bill. If the field is empty, we recommend you select the vendor invoice
number as a reference.
```
Adjusted for 100 char/line

Upon payment of the vendor bill, click on :guilabel:`Register Payment`. A new window pops up.

Select the :guilabel:`Journal`, the :guilabel:`Payment Method`, the :guilabel:`Amount` you wish to

> pay (full or partial payment), and the :guilabel:`Currency`. In the case of a partial payment (when
> the :guilabel:`Amount` paid is less than the vendor invoice amount), a :guilabel:`Payment Difference`
> field is available to :guilabel:`Keep open` (marked with a :guilabel:`Partial` banner) or
> :guilabel:`Mark as fully paid`.

Reviewer 2: ```suggestion
pay (full or partial payment), and the :guilabel:`Currency`. In the case of a partial payment (when
the :guilabel:`Amount` paid is less than the total remaining amount on the vendor bill), the :guilabel:`Payment Difference` field displays the outstanding balance. You have two options:
- :guilabel:`Keep open`: to keep the bill open and mark it with a :guilabel:`Partial` banner;
- :guilabel:`Mark as fully paid`: In this case, select a chart of accounts in the :guilabel:`Post Difference In` field and change the :guilabel:`Label` if needed. 
```
- To clarify the part about the vendor invoice amount (I also prefer to stick with "bill" for consistency here)
- To mention the fact that the payment difference is automatically computed and displayed
- I am by no means an accounting expert, but I noticed that 2 additional fields were revealed if you select "Mark as fully paid". It might be helpful to clarify (very briefly) *why* you need to specify the chart of accounts but I'm not 100% sure it's necessary --> if it is, I would just add something like "select a chart of accounts to XXX in the :guilabel:`Post Difference In` field". 

![image](https://github.com/user-attachments/assets/4ba678c2-31bf-4ede-ba9e-79ff7be4f45a)

Reviewer: Jumping in here uninvited and I hope that's okay :)
- I support sticking with "vendor bill" and "customer invoice" so it's always clear what we're talking about (though if it's getting repetitive, I'm okay shortening them to "invoice" and "bill" respectively, but they shouldn't be mixed with each other)
- Yes, a brief explanation as to why you need to choose an account like Audrey suggested would be great, but you want to select an **account** not a chart of accounts. The chart of accounts is just the list of all of your accounts. Consider this but feel free to make it your own: "- :guilabel:`Mark as fully paid`: In this case, select an account in the :guilabel:`Post Difference In` field and add a :guilabel:`Label` if needed. Odoo will create a journal entry balancing the account receivable with whichever account is selected." (Note that the future tense "Odoo will create" is intentional because it _will_ do it once you click on the button.

Author: @auva-odoo Thanks for the comments! I made the changes, but I have some questions ;) I'll check it with you on Thursday.
@dade-odoo Thank you for your comments too ;) I noticed the 'chart of accounts' vs 'account' so I've already fixed it. I'll also add the last sentence. Great info, thanks!
